### PR TITLE
Disable mob drop multiplication for horses

### PR DESCRIPTION
**Pull request by TealNerd**

### CODE REVIEW

**Comment by ttk2** [comment]
thank you, also Donkeys?

**Comment by TealNerd** [comment]
donkeys are horses and bukkit is stupid

**Comment by ttk2** [comment]
that's been established.

**Comment by ttk2** [comment]
BTW main server drop rate is 1 until this goes online tomorrow.

**Comment by TealNerd** [comment]
but yeah you should merge this so it can build

**Comment by ttk2** [comment]
going to have someone do a quick review, looks good to me though, but I know nothing about humbug

**Comment by TealNerd** [comment]
its just the one line that checks if the mob is a player so it doesnt dupe player drops. i added horses to the if statement

**Comment by Pentom** [comment]
Code Review:  http://www.reddit.com/r/Civcraft/comments/32bqoe/morning_changelog_20150412/cq9pvxw

TLDR:  Feature loses a miniscule bit of functionality that it had before (previoulsy it wiped out horse mob xp drops but now it allows them).  It has no noticeable affect on civcraft since xp is disabled (validated explicitly and stated in review).  I've suggested that this functionality be either reintroduced for horses to keep what we have now or removed.  I would not necessarily push for this as a pre-req for acceptance of this pull though since I think adding the feature back in for horses is stupid since I would prefer it to be gone for all mob types.  I've added issue 22 to track this bad configuration option so this pull -can- be accepted without any changes as long as we accept that horses will drop useless xp orbs.  https://github.com/Civcraft/Humbug/issues/22 

TLDR TLDR:  It appears fine to accept with a minor useless problem that may or may not be fixed.

**Comment by ttk2** [comment]
Erocs just set the multiplier for horses specifically to 1 seems to work
for now. But it makes cases where other people using this feature could
face unintended consequences. Suggested correction involves having the
horse multiplier continue to exist but not fall under general. Or we could
actually differentiate between user stuff on a horse and its drops.

**Comment by rourke750** [comment]
Does this need to be merged?

**Comment by ttk2** [comment]
I think erocs just set the multiplier to one manually, it perhaps should be
that way by default instead of having a hard coded override.
